exporters/otlp/otlplog/otlploghttp/exporter_test.go

```diff
@@ -34,3 +37,41 @@ func TestExporterForceFlush(t *testing.T) {
 
 	assert.NoError(t, e.ForceFlush(ctx), "ForceFlush")
 }
+
+func TestExporterConcurrentSafe(t *testing.T) {
+	ctx := context.Background()
+	e, err := New(ctx)
+	require.NoError(t, err, "newExporter")
+
+	const goroutines = 10
+
+	var wg sync.WaitGroup
+	ctx, cancel := context.WithCancel(context.Background())
+	runs := new(uint64)
+	for i := 0; i < goroutines; i++ {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+
+			r := make([]log.Record, 1)
+			for {
+				select {
+				case <-ctx.Done():
+					return
+				default:
+					atomic.AddUint64(runs, 1)
+					_ = e.Export(ctx, r)
+					_ = e.ForceFlush(ctx)
+				}
+			}
+		}()
+	}
+
+	for atomic.LoadUint64(runs) == 0 {
+		runtime.Gosched()
+	}
```

Comment:
If we want to make sure that any `Export` and `ForceFlush` is called before `Shutdown`.

```suggestion
	var wg sync.WaitGroup
	ctx, cancel := context.WithCancel(context.Background())
	called := make(chan struct{}, goroutines)
	for i := 0; i < goroutines; i++ {
		wg.Add(1)
		go func() {
			defer wg.Done()

			r := make([]log.Record, 1)
			for {
				select {
				case <-ctx.Done():
					return
				default:
					_ = e.Export(ctx, r)
					_ = e.ForceFlush(ctx)
					called <- struct{}{}
				}
			}
		}()
	}

	<- called
```